transformations.yml

```diff
@@ -26,6 +26,7 @@ transformations:
       - "*/**.go"
       - "go.mod"
       - "go.sum"
+      - "Makefile"
```

Comment:
Note: we also need to transform the Makefile since `ldflags` now references `feather-core` variables. This can be avoided if we publish a separate package for these variables, then import that package in (i.e. decouple where the config variables live). However, this may be overly complicated for our current needs.